velox/exec/GroupingSet.cpp

```diff
@@ -598,6 +606,73 @@ bool GroupingSet::getGlobalAggregationOutput(
   return true;
 }
 
+bool GroupingSet::getDefaultGlobalGroupingSetOutput(
+    int32_t batchSize,
```

Comment:
What is batchSize here?

Reply:
HashAggregation invokes groupintSet_->getOutput() with the number of output rows which it sets as 1 for global aggregation. Else it sends queryConfig.preferredOutputBatchRows(). I could add the case of globalGroupingSets as well in the HashAggregation but that seemed out of place logic.

Reply:
Looks like this parameter is not needed. Let's remove. I'll send out a PR.